client/landing/stepper/declarative-flow/internals/steps-repository/import/helper.ts

```diff
@@ -75,3 +75,46 @@ export function generateStepPath( stepName: string, stepSectionName?: string ) {
 		}
 	}
 }
+
+/**
+ * Returns true if the platform is importable
+ *
+ * @param platform - The platform to check
+ * @returns True if the platform is importable, false otherwise
+ */
+export function isPlatformImportable( platform: ImporterPlatform ) {
+	const productImporters = getImporterEngines();
+	return productImporters.includes( platform );
+}
+
+/**
+ * Returns the full importer URL for the given platform
+ *
+ * @param platform - The platform to get the importer URL for
+ * @param targetSlug - The target slug for the importer URL
+ * @param fromSite - The from site for the importer URL
+ */
+export function getFullImporterUrl(
+	platform: ImporterPlatform,
+	targetSlug: string,
+	fromSite: string
+) {
+	if ( ! isPlatformImportable( platform ) ) {
+		return getWpOrgImporterUrl( targetSlug, platform );
+	}
+
+	const hasSiteSetupImporter = [ 'blogger', 'medium', 'squarespace', 'wix', 'wordpress' ];
```

Comment:
hardcoding this list doesn't feel OK. What about adding an extra property in the importer list: `https://github.com/Automattic/wp-calypso/blob/1997d0e7f0ab7801a4879b7098f5d0f858f9771a/client/lib/importer/importer-config.tsx#L54`.
My reasoning is that if we add an importer, there would be no way of knowing that we should update this array.